README.md

```diff
@@ -6,7 +6,7 @@ Passed the VITA exams with top-ranked scores, I worked as an volunteer income ta
 
 Previously, as a staff accountant at GF Securities Co., Ltd., I maintained ledgers and compiled company financial statements. My responsibilities include invoice processing, document coordination, revamping systems for accuracy and efficiency, and resolving discrepancies and payment issues promptly. I initiated an income and expense flux analysis for the first working day of the month that provides for early identification and correction of posting errors, and contributed to accelerating the month-end close to the twentieth calendar day of every month. 
 
-In an academic project, I analyzed annual reports of American Eagle and Urban Outfitters. By computing over ten crucial finance indicators, I evaluated the profit and operation stability and predicted stock price trends in the following two years.
+In an academic project, I analyzed annual reports of American Eagle and Urban Outfitters. By computing over ten crucial finance indicators, I evaluated the profit and operation stability and predicted stock price trends in the following two years. For exampele, return of assets (ROA) of American Eagle and Urban Ourfitters is 4.74% and 11.31% respectively, compared to the industry average level is 8.68% for the year 2015. It is safe to say that based on this indicator alone, Urban Outfitters has better performance, since higher ROA means the company is earning more profit on less investment.
```

Comment:
Including only data showing off your ability and contribution; these are irrelevant to your job application --- did you contribute to Urban Outfitters' high ROA of 11.31%? I don't think so. So why would a recruiter care about whichever of American Eagle and Urban Outfitters won?

As an example of data that show your capacity includes: now that you mentioned you "predicted stock price trends," did you check your prediction against the real trends? How accurate was it? Also, you "evaluated the profit and op stability," was this evaluation universal, i.e., would a different person conclude differently? If so, is there any indicator on how good your evaluation was?